styles/_use-cases.scss

```diff
@@ -30,13 +95,422 @@
   font-size: 1rem;
 }
 
+.arrows-container {
+  position: absolute;
+  top: 50%;
+  left: 0;
+  right: 0;
+  transform: translateY(-50%);
+  display: flex;
+  justify-content: space-between;
+  z-index: 10;
+  @media only screen and (max-width: 768px) {
+    top: 30px;
+  }
+}
+
+.arrow-button {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+}
+
+.arrow-button img {
+  width: 40px;
+  height: 40px;
+}
+
+.left-arrow {
+  margin-left: 40px;
+  @media only screen and (max-width: 768px) {
+    margin-left: 0px;
+  }
+}
+
+.right-arrow {
+  margin-right: 40px;
+  @media only screen and (max-width: 768px) {
+    margin-right: 0px;
+  }
+}
+
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1000;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(0, 0, 0, 0.4);
+}
```

Comment:
A lot of this is redefining / overriding [Bootstrap `modal` styles](https://getbootstrap.com/docs/4.1/components/modal/) that already exist. Can we simplify by using the existing Bootstrap modals?

Reply:
I changed the classname from .modal to .modal-uses to avoid any conflicts with bootstrap's modal styles